tests/FieldHelper/RichTextFieldHelperTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Elbformat\FieldHelperBundle\Tests\FieldHelper;
+
+use Elbformat\FieldHelperBundle\FieldHelper\RichTextFieldHelper;
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\Core\FieldType\RichText\Value as RichTextValue;
+use eZ\Publish\Core\FieldType\Value;
+use eZ\Publish\Core\Helper\FieldHelper;
+use eZ\Publish\Core\Repository\Values\Content\Content;
+use eZ\Publish\Core\Repository\Values\Content\ContentUpdateStruct;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author Ronny Gericke <ronny.gericke@elbformat.de>
+ */
+class RichTextFieldHelperTest extends TestCase
+{
+    protected RichTextFieldHelper $richtextFieldHelper;
+
+    public function setUp(): void
+    {
+        $fieldHelper = $this->createMock(FieldHelper::class);
+
+        $this->richtextFieldHelper = new RichTextFieldHelper($fieldHelper);
+    }
+
+    public function testGetName(): void
+    {
+        $this->assertSame('Elbformat\FieldHelperBundle\FieldHelper\RichTextFieldHelper', RichTextFieldHelper::getName());
+    }
+
+    public function testGetXml(): void
+    {
+        $value = $this->getXmlDomFromString('some text');
+        $content = $this->createContentFromValue($value);
+
+        $this->assertSame(
+            $this->richtextFieldHelper->getXml($content),
+            $value,
+        );
+    }
+
+    /**
+     * @dataProvider getUpdateValues
+     */
+    public function testUpdateXml(?\DOMDocument $newValue, bool $expectedResult): void
+    {
+        $struct = new ContentUpdateStruct();
+        $value = new RichTextValue(
+            $this->getXmlDomFromString('initial value')
+        );
+        $content = $this->createContentFromValue($value);
+
+        $this->assertSame(
+            $this->richtextFieldHelper->updateXml($struct, 'rich_text_field', $newValue, $content),
```

Comment:
The first parameter should be the expected value. Otherwise the error message is irritating.